aptos-move/aptos-vm/src/natives.rs

```diff
@@ -91,4 +94,9 @@ fn unit_test_extensions_hook(exts: &mut NativeContextExtensions) {
     exts.add(NativeTransactionContext::new(vec![1], ChainId::test().id())); // We use the testing environment chain ID here
     exts.add(NativeAggregatorContext::new([0; 32], &*DUMMY_RESOLVER));
     exts.add(NativeRistrettoPointContext::new());
+    let mut features = Features::default();
```

Comment:
I think we'll want to expose the features from `aptos_framework::features` via an `OnChainConfig` in `types/src/on_chain_config/aptos_features.rs`.

This is because we'll want to decide whether a new curve is enabled via a governance proposal that updates those flags in `aptos_framework::features` (as opposed to the feature being manually enabled in the Rust code).

Reply:
Otherwise, the features you declared in `features.move` will not be picked up here, no?

Reply:
In other words, you are not reading the features here from the chain (which might've or might've not been enabled). You are just enabling them manually.

Reply:
e.g., you have to pass them in from [here](https://github.com/aptos-labs/aptos-core/blob/eef180dce06aab95f84ca02ed4db9dff3a39beeb/aptos-move/aptos-vm/src/aptos_vm_impl.rs#L131) (where they are read)

Reply:
Also it's sort of in place already: earlier I made a change to read the entire feature flag vector and pass it to the move vm [here](https://github.com/aptos-labs/aptos-core/blob/b006f31f276feb3b45d924f3d749a0250e6363ba/aptos-move/aptos-vm/src/aptos_vm_impl.rs#L149).

Further passing them to extensions will be easy.